bmv2/Makefile.am

```diff
@@ -29,6 +29,35 @@ $(builddir)/@P4_NAME@/thrift-src/pd_rpc_server.h
 
 P4_JSON_OUTPUT = $(P4_NAME).json
 
+plugin_files =
+plugin_headers =
+
+# Openflow plugin
+if COND_PLUGIN_OPENFLOW
+of_sources = \
+@P4_NAME@/plugin/of/src/pd_wrappers.c    \
+@P4_NAME@/plugin/of/src/ofpat_groups.c   \
+@P4_NAME@/plugin/of/src/ofpat_state.c    \
+@P4_NAME@/plugin/of/src/ofpat_pipeline.c
+
+of_includes = \
+@P4_NAME@/plugin/of/inc/pd_wrappers.h    \
+@P4_NAME@/plugin/of/inc/ofpat_groups.h   \
+@P4_NAME@/plugin/of/src/ofpat_state.h    \
+@P4_NAME@/plugin/of/inc/ofpat_pipeline.h \
+@P4_NAME@/plugin/of/inc/openflow.h
+
+plugin_files += $(of_sources) $(of_includes)
+plugin_headers += $(of_includes)
```

Comment:
if I am not mistaken, these should be installed, i.e. listed in a `_HEADERS` variable.
For example:
```
ofincludedir = $(includedir)/bmpd/@P4_NAME@/of
ofinclude_HEADERS = $(pd_headers)
```
Also if `ofpat_state.h` is an internal header (it it in the `src` directory), it should not be listed with the public headers but with the source files.